@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/headers/core.hpp

```diff
@@ -37,6 +37,6 @@
 
 #define HDG_HOLD_SPEED_SWITCH_DECEL   15.433  //30kts GS
 #define HDG_HOLD_SPEED_SWITCH_ACCEL   20.577  //40kts GS
-#define HDG_HOLD_BREAKOUT_VALUE       0.03
-#define VEL_HOLD_BREAKOUT_VALUE       0.06
-#define ATT_HOLD_BREAKOUT_VALUE       0.09
\ No newline at end of file
+#define HDG_HOLD_BREAKOUT_VALUE       0.030
```

Comment:
Why did you add the suffixing 0?

Reply:
3 digit precision was used to allow for 2.5%, was updated to 3, the 3 digit precision remained